modules/iam-policy/ecr-create-repository/providers.tf

```diff
@@ -15,13 +15,13 @@ provider "aws" {
   secret_key = var.secret_key
   region     = var.region
 
-  # TODO
   default_tags {
     tags = {
-      "managed-by" = "humanitec"
-      "hum-app-id" = var.app_id
-      "hum-env-id" = var.env_id
-      "hum-res-id" = var.res_id
+      "humanitec"  = "true"
+      "hum:app"    = var.app_id
+      "hum:env"    = var.env_id
+      "hum:res"    = var.res_id
+      "managed_by" = "terraform"
```

Comment:
Those are what I came up with looking at what is currently used:
<img width="375" alt="image" src="https://github.com/humanitec-architecture/resource-pack-aws/assets/864578/1732319c-4b1d-46bf-aca3-d9b802f2609b">